src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -1372,9 +1379,10 @@ private native long map0(int prot, long position, long length, boolean isSync)
     // Removes an existing mapping
     private static native int unmap0(long address, long length);
 
-    // Transfers from src to dst, or returns -2 if kernel can't do that
-    private native long transferTo0(FileDescriptor src, long position,
-                                    long count, FileDescriptor dst);
+    // Transfers from src to dst, or returns IOStatus.UNSUPPORTED (-4)
+    // or IOStatus.UNSUPPORTED_CASE (-6) if kernel can't do that
+    private static native long transferTo0(FileDescriptor src, long position,
```

Comment:
It might be better to say "if the kernel does not support it" rather than "kernel can't do that".

Reply:
Agreed. That phrase was already there. I will change it.